frontend/app/utils/string-utils.ts

```diff
@@ -37,3 +39,97 @@ export function padWithZero(value: number, maxLength: number): string {
 export function trimToUndefined(str: string | undefined): string | undefined {
   return Number(str?.trim().length) > 0 ? str : undefined;
 }
+
+/**
+ * Arguments for formatting just the address line
+ */
+interface FormatAddressLineArguments {
+  /** Street address */
+  address: string;
+  /** Apartment, suite, or unit number */
+  apartment?: string;
+}
+
+/**
+ * Formats an apartment/suite number with the address
+ * @param params Address line formatting parameters
+ * @returns Formatted address line
+ */
+export function formatAddressLine({ address, apartment }: FormatAddressLineArguments): string {
```

Comment:
I don't think this needs to be exported.. if you wanted to only format a street name, wouldn't you just be able to call: `formatAddress({ address })`?

I also think this should be named `formatStreet()` and the function signature should be: `formatStreet(streetName: string, apartment?: string)`. I don't see any reason to wrap the parameters in an object.

Reply:
Also.. what if the user enters something like `123 A` as the apartment number instead of `123a`? This would render `Anywhere Street 123 A`, when it should be `123A Anywhere Street`.

Sadly, I don't know if there is any way to avoid this.